tools/Azure.Mcp.Tools.Monitor/src/Commands/Log/ResourceLogQueryCommand.cs

```diff
@@ -18,16 +18,12 @@ public sealed class ResourceLogQueryCommand(ILogger<ResourceLogQueryCommand> log
 
     public override string Description =>
         $"""
-        Executes a Kusto Query Language (KQL) query to retrieve logs for any Azure resource that emits logs to Log Analytics.
-
+        Queries logs for a specific Azure resource using Kusto Query Language (KQL) from a Log Analytics workspace. Use this tool to retrieve logs for individual resources such as storage accounts, virtual machines, app services, or any resource that emits logs to Log Analytics. You can also query by resource ID to investigate diagnostics, errors, and activity logs.
```

Comment:
I think we can get even better separation by removing workspace from the first sentence. You don't always need a workspace to get logs from a resource.

We can try the following or some variant of it:

```suggestion
        Queries logs from Log Analytics for a specific Azure resource using Kusto Query Language (KQL). Use this tool to retrieve logs for individual resources such as storage accounts, virtual machines, app services, or any resource that emits logs to Log Analytics. You can also query by resource ID to investigate diagnostics, errors, and activity logs.
```